### PR TITLE
Contract test

### PR DESCRIPTION
**Pull request by R4il**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-RUN mkdir -p /go/src/github.com/AITestingOrg/calculation-service
-WORKDIR /go/src/github.com/AITestingOrg/calculation-service
+RUN go env GOPATH
+RUN mkdir -p /go/src/calculation-service
+WORKDIR /go/src/calculation-service
 
 COPY . .
 
-RUN go get -v
+RUN go get -v ./...
```
Comment (Castyr) [inline Dockerfile:10]: this looks terrible

File: tests/provider_test.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
+// +build !contract
+
+package tests
+
+// import (
```
Comment (Castyr) [inline tests/provider_test.go:5]: why is this entire thing commented out?

**Review by Castyr — APPROVED** [review]

File: tests/provider_test.go (hunk 2 above)
Comment (R4il) [reply tests/provider_test.go:5]: There is no provider test ready yet, so this cant run, but it was running on the wrong group and failing so I took it out, but I want to keep the skeleton.

File: Dockerfile (hunk 1 above)
Comment (R4il) [reply Dockerfile:10]: the ./... ? is the go way of saying go and search fully recursively.

**Pull request merged**